### PR TITLE
Add account setup checklist

### PR DESCRIPTION
**Pull request by peyerluk**
## Changelog

- 🎁 Add a checklist for which accounts and services to configure when moving past the evaluation in the 'On Premise Setup' document.

### CODE REVIEW

**Comment by peyerluk** [comment]
I noticed that we do not have documentation on what to do with the Boilerplate configuration when starting to develop. And as almost all customers start from the Boilerplate I wanted to add instructions.

I'm not sure if it's the right place though and if some things might still be missing.

File: content/learn/on-premise-setup.md [hunk 1]
```diff
@@ -77,10 +77,62 @@ export ENVIRONMENT=local
...
+
+Server Config Keys to update:
+  * `notifications.channels.email.fromAddress`
+  * `emails.transport.default.*`
+
+#### NPM Access Token
+
+To access private livingdocs repositories you need to create a npm user with an access token. We can then grant your npm user read access to our @livingdocs packages (e.g. `@livingdocs/livingdocs-server` and `@livingdocs/livingdocs-editor`).
+
+We then recommend to add an `.npmrc` file to your server and editor repositories so everyone in your team can run `npm install` seamlessly.
```
Comment (DaRaFF) [inline content/learn/on-premise-setup.md:138]: We recommend adding a `.npmrc` file to your server and editor repositories so everyone in your team can run `npm install` seamlessly.

```bash
# .npmrc
//registry.npmjs.org/:_authToken=<yourToken>
```

**Review by DaRaFF — APPROVED** [review]
nice, I'm sure this will help the customers 👍

**Comment by DaRaFF** [comment]
> I'm not sure if it's the right place though and if some things might still be missing.

I was going through the current boilerplate quickly and detected additional accounts which need to be changed when going on-premise seriously.
- iframely (editor config)
- tracks (editor config)

**Pull request merged**